torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -115,7 +115,9 @@ std::pair<IValue, c10::optional<IValue>> getFunctionTuple(
 
   Inline(*graph);
 
-  torch::jit::Code code(graph, func.name());
+  // TODO: to use it for mobile, you need to enable it here.
```

Comment:
Is this statement up to date? I'm assuming I should use MobileCode?

Reply:
Yes sorry the TODO should have been removed. Basically emitOperator method in MobileCode is disabled right now sych that it calls emitOperator from its' parent class which is Code.